src/Websites/Websites/help/New-AzWebAppManagedCertificate.md

```diff
@@ -0,0 +1,163 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Websites.dll-Help.xml
+Module Name: Az.Websites
+online version:https://docs.microsoft.com/en-us/powershell/module/az.websites/new-azwebappmanagedcertificate
+schema: 2.0.0
+---
+
+# New-AzWebAppManagedCertificate
+
+## SYNOPSIS
+Creates an App service managed certificate for an Azure Web App. 
+
+## SYNTAX
+
+```
+New-AzWebAppManagedCertificate [-ResourceGroupName] <String> [-WebAppName] <String> [[-Slot] <String>]
```

Comment:
I thought we discussed ant providing an optional parameter for certificate Name?

Reply:
Yes Sisira, after my analysis we had one more discussion that the even we provide `Name` in request object, the API is considering `CanonicalName` as certificate name.